### ISSUE TITLE
Question about security

### ISSUE THREAD

**Issue by lesharadkovich** [mCodex/react-native-sensitive-info#97]
Hi all. Sorry for my possibly silly question, cause I don't really know much about protecting data
I'm intrested in how secure is this lib (from keystore branch)? can someone get access to data from another app?
and one more question: is this lib better in security than for example [react-native-sqlcipher](https://github.com/sreejithkrishnanr/react-native-sqlcipher-2)?

**Comment by zedtux** [comment]
Yep, no recent activity, but from @mCodex.

**Comment by mCodex** [comment]
Hi,

```
Yep, no recent activity, but from @mCodex.
```

**Comment by mCodex** [comment]
Hi,

```
Yep, no recent activity, but from @mCodex.
```

First of all, I'm sorry about that as I stated before, I wasn't actively maintaining this project I was just merging and releasing new versions. However, since yesterday I came back and started organizing this repo.


```
Hi all. Sorry for my possibly silly question, cause I don't really know much about protecting data
I'm intrested in how secure is this lib (from keystore branch)? can someone get access to data from another app?
and one more question: is this lib better in security than for example react-native-sqlcipher?
```

@lesharadkovich it's a not silly question 😄don't worry.

Keystore and Keychain are system secured storage for each platform used by many applications . I really don't know much about SQLCipher and how it works internally. So, I can't answer which lib is more secure than other (if it exists)

About your data being accessed by another app, by default it can't because keychain and keystore doesn't allow it. If you want that, your apps must belong to the same group then they can share data among them. On the other hand, if the user's phone is rooted I can't tell how it works and if the data can be easily decrypted I don't think so.
